Offseason2022/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -157,98 +157,98 @@ private void addSmartDashboardWidgets( )
+    //     new AutoDriveLimelightShoot(m_drivetrain, m_intake, m_floorConveyor, m_towerConveyor, m_shooter, m_vision));
+    // SmartDashboard.putData("AutoDrivePath", new AutoDrivePath(m_drivetrain, "simCurvePath", true));
     SmartDashboard.putData("AutoDriveShoot",
         new AutoDriveShoot(m_drivetrain, m_intake, m_floorConveyor, m_towerConveyor, m_shooter, m_vision));
     SmartDashboard.putData("AutoPathSequence", new AutoPathSequence(m_drivetrain));
-    SmartDashboard.putData("AutoShoot",
-        new AutoShoot(m_drivetrain, m_intake, m_floorConveyor, m_towerConveyor, m_shooter, m_vision));
-    SmartDashboard.putData("AutoShootDriveShoot",
-        new AutoShootDriveShoot(m_drivetrain, m_intake, m_floorConveyor, m_towerConveyor, m_shooter, m_vision));
-    SmartDashboard.putData("AutoShootLowHub",
-        new AutoShootLowHub(m_drivetrain, m_intake, m_floorConveyor, m_towerConveyor, m_shooter, m_vision));
-    SmartDashboard.putData("AutoStop", new AutoStop(m_drivetrain));
-
-    SmartDashboard.putData("Climber0Stow",
-        new Climber0Stow(m_climber, m_intake, m_floorConveyor, m_towerConveyor, m_shooter, m_drivetrain));
-    SmartDashboard.putData("Climber1Deploy",
-        new Climber1Deploy(m_climber, m_intake, m_floorConveyor, m_towerConveyor, m_shooter, m_drivetrain));
-    SmartDashboard.putData("Climber2ClimbToL2", new Climber2ClimbToL2(m_climber));
-    SmartDashboard.putData("Climber3RotateToL3", new Climber3RotateToL3(m_climber));
-    SmartDashboard.putData("Climber5RotateIntoL3", new Climber5RotateIntoL3(m_climber));
-    SmartDashboard.putData("Climber6ClimbToL3", new Climber6ClimbToL3(m_climber));
-    SmartDashboard.putData("Climber7ClimbToL4", new Climber7ClimbToL4(m_climber));
-    SmartDashboard.putData("Climber8SettleToL4", new Climber8SettleToL4(m_climber));
-    SmartDashboard.putData("ClimberCalibrate", new ClimberCalibrate(m_climber));
-    SmartDashboard.putData("ClimberFullClimb", new ClimberFullClimb(m_climber, m_operator, XboxController.Button.kY));
-    SmartDashboard.putData("ClimberL2ToL3", new ClimberL2ToL3(m_climber, m_intake, m_floorConveyor, m_towerConveyor, m_shooter));
-    SmartDashboard.putData("ClimberL3ToL4", new ClimberL3ToL4(m_climber, m_intake, m_floorConveyor, m_towerConveyor, m_shooter));
-    SmartDashboard.putData("ClimberRun", new ClimberRun(m_climber, m_operator));
+    // SmartDashboard.putData("AutoShoot",
+    //     new AutoShoot(m_drivetrain, m_intake, m_floorConveyor, m_towerConveyor, m_shooter, m_vision));
+    // SmartDashboard.putData("AutoShootDriveShoot",
+    //     new AutoShootDriveShoot(m_drivetrain, m_intake, m_floorConveyor, m_towerConveyor, m_shooter, m_vision));
+    // SmartDashboard.putData("AutoShootLowHub",
+    //     new AutoShootLowHub(m_drivetrain, m_intake, m_floorConveyor, m_towerConveyor, m_shooter, m_vision));
+    // SmartDashboard.putData("AutoStop", new AutoStop(m_drivetrain));
+
+    // SmartDashboard.putData("Climber0Stow",

```

Comment:
Let's leave climber deploy and climber stow commands uncommented so they can be run in simulation easily.